readthedocs/builds/forms.py

```diff
@@ -31,8 +31,8 @@ class Meta:
         model = Version
         fields = ['active', 'privacy_level', 'tags']
 
-    def save(self, *args, **kwargs):
-        obj = super(VersionForm, self).save(*args, **kwargs)
+    def save(self, commit=True):
+        obj = super(VersionForm, self).save(commit=commit)
         if obj.active and not obj.built and not obj.uploaded:
             trigger_build(project=obj.project, version=obj)
         return obj
```

Comment:
I do worry that this class of change will have spiraling effects that only show up in run time. Is there a reason we don't want to continue accepting args & kwargs for these things?